app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -13,7 +13,7 @@ import { Link } from 'react-router';
 import CommentView from 'app/components/Comments/CommentView';
 import Modal from 'app/components/Modal';
 import styles from './GalleryPictureModal.css';
-import Swipeable from 'react-swipeable';
+import { Swipeable, RIGHT, LEFT } from 'react-swipeable';
```

Comment:
Should these be imported as types, or?

Reply:
Is there a way to get these types in flow?
![image](https://user-images.githubusercontent.com/23152018/62832238-928f6f00-bc2b-11e9-8d85-855fd2306e71.png)

or should one just `import type {RIGHT, LEFT} from 'react-swipeable'`?
![image](https://user-images.githubusercontent.com/23152018/62832254-c8ccee80-bc2b-11e9-874b-99a449e18e51.png)

Reply:
The current import looks correct, since the values are ja exports, not types :+1: